source/index.html.md

```diff
@@ -686,7 +686,8 @@ object][ApplePay-PaymentToken] for more information.
     <span class="type">[0-9]{12,19}</span>
   </dt>
   <dd>
-    Primary account number of card to charge.
+    Primary account number of card to charge.<br />
+    If <code>token_callback</code> parameter is included, the <code>pan</code> must be a Token PAN.
```

Comment:
Is "token callback" a phrase used by both Visa and Mastercard? I think the current state of this PR suggests to the reader that it is.

Reply:
I would say that `tokenCallback` is the closest MPO comes to giving it a name. It is the entire _thing_ returned from the tokenization service. 

https://github.com/MobilePayDev/MobilePay-Online/pull/25/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R101https://github.com/MobilePayDev/MobilePay-Online/pull/25/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R101

> When you recieve the tokenCallback, you´ll find a cardIssuedInCountryCode (possible values DK, FI) you can use for your Acquirer routing logic. And a Visa Token Service (VTS) service response like this: <br />

I have purposely used MPO terminology to avoid having to pick between Visa or Mastercard terminology.

But the naming link in flimsy, and a better option would be preferable. Golf away! :golfing:

Reply:
Fair :slightly_smiling_face: It seems, however, `tokenCallback` is only mentioned in the "Delegated Authentication for all Visa cards" section, not in the "Delegated Authentication for all Mastercards" section. It will be interesting to see what happens w.r.t. Mastercard support. It is likely a favorable bet that it will also involve a `tokenCallback`, as they're also aiming for tokenization with Mastercard and state that both tokenization schemes are based on EMVCo.

Reply:
Let us settle on a parameter name: Currently: `token_callback`. Solely based on

> When you recieve the tokenCallback, you´ll find a cardIssuedInCountryCode (possible values DK, FI) you can use for your Acquirer routing logic.

Which is not good.
Just `token` is a bit to generic and could be interpreted as just the `tokenData` part of the data received on tokenCallback. Alternatively `token_callback_data`.

Finally we can just go with `payment_token` just like `applepay`.

Reply:
> Finally we can just go with `payment_token` just like `applepay`.

I like the strategy and think `payment_token` is a decent name. 👍